eclair-core/src/test/scala/fr/acinq/eclair/channel/states/a/WaitForOpenChannelStateSpec.scala

```diff
@@ -177,6 +193,15 @@ class WaitForOpenChannelStateSpec extends TestkitBaseClass with StateTestsHelper
     awaitCond(bob.stateName == CLOSED)
   }
 
+  test("recv OpenChannel (wumbo size)", Tag("wumbo"), Tag("max-funding-satoshis")) { f =>
+    import f._
+    val open = alice2bob.expectMsgType[OpenChannel]
+    val highFundingSat = Btc(1).toSatoshi
+    bob ! open.copy(fundingSatoshis = highFundingSat)
+    bob2alice.expectMsgType[AcceptChannel]
```

Comment:
I think we're not yet testing that when we accept a wumbo channel, we correctly set the `min-depth` in our accept message to something that scaled on the amount of the channel.

Maybe here we should use something bigger than 1 BTC (that should have a min-depth > 6) and assert that the `AcceptChannel` sent by Bob correctly sets that `min-depth`?

Reply:
Uhm I believe that's done in [`WaitForAcceptChannelStateSpec`](https://github.com/ACINQ/eclair/pull/1323/files#diff-42debadc7a9bf325bd7ceaa8545be205R162)? Here we do test the usage of `min_depth` in `AcceptChannel` and raw invocation of `minDepthForFunding`

Reply:
Not really because this test expects the result to be the default conf value of 6.
We can't know that the scaling factor was applied and then rounded to 6 because it was lower than that.
If you change that test to use a funding of 5 BTC (and thus min-depth of 20) then that would work.

Reply:
But I agree that the test you're pointing to is likely the best place to test this.

Reply:
Okay :), i was using 1BTC in the test to check the "maxing" with the configured value, i changed it now to 5BTC and added the test for the configured value right below it (note that by our formula 5BTC -> 13 confs). Do you want me to squash the commit history of this PR for easier review? It has reached a whopping 10 commits already..